### PR TITLE
Add JSON syntax colors and fix Airline tabline colors.

### PR DESCRIPTION
**Pull request by jinh0**
There is no JSON syntax rules so the quotation marks would come out white half the time and purple half the time.

I decided to make the quotation marks green to match the keyword color. I couldn't find a way to have different quotation mark colors for the keyword and the value respectively, but I think this turned out good.

Before:
Screen Shot 2021-04-08 at 5 55 19 PM

After:
Screen Shot 2021-04-08 at 6 16 58 PM

### CODE REVIEW

**Comment by pineapplegiant** [comment]
Let's have you add the stuff you did for the tabline to this PR since you're doing this too.

Also perhaps change the quotation color to blue instead to match the other strings? What do you think?

Screen Shot 2021-04-08 at 8 25 14 PM

File: estilo/syntax/base.yml [hunk 1]
```diff
@@ -48,9 +48,9 @@ StatusLine: 'foreground background r'
 StatusLineTerm: '@StatusLine'
 StatusLineTermNC: 'pureBlack darkPurp r'
 StatusLineNC: '@StatusLineTermNC'
-TabLine: 'pureBlack grey'
-TabLineFill: 'grey pureBlack'
-TabLineSel: 'grey2 background'
+TabLine: 'foreground background'
+TabLineFill: 'darkPurp2 background'
+TabLineSel: 'purple background'
```
Comment (jinh0) [inline estilo/syntax/base.yml:53]: There isn't anything in Estilo for Airline tabline, but in `syntax/base.yml`, these three lines seem to be incorrect so I matched it according to the new changes, but to be 100% transparent, I have no idea how to use Estilo.

**Comment by jinh0** [comment]
I added the Airline tabline colors as well in `autoload/airline/themes/spaceduck.vim`.

Regarding adding to the Estilo Airline file, [there seems to be no options for the tabline](http://estilo.jacoborus.codes/airline.html). I hope that's not an issue.

**Comment by jinh0** [comment]
Regarding the quotation marks, I originally tried out blue, like you suggested, but I found it doesn't go as smoothly with the keywords (blue on left, green on right):
Screen Shot 2021-04-09 at 7 33 37 PM

Yes, blue is more consistent with strings in other file types. It's a mere preference to have it green to match the keywords rather than the values. Sadly JSON files don't allow keywords without quotes like JavaScript, so it's quite annoying. It's up to you.

File: estilo/syntax/base.yml (hunk 1 above)
Comment (pineapplegiant) [reply estilo/syntax/base.yml:53]: I’ll test it no worries. I’m gonna deprecate it soon as it’s causing me more pain then anything else these days.

**Comment by pineapplegiant** [comment]
Let’s stick with the green like you had, you’re right it looks better in JSON this way.

**Comment by pineapplegiant** [comment]
The only question I have is do you know if we need to edit the airline tabline or will it source those changes by default from the base color scheme? 

The only reason I ask is because I’ve taken a quick look at other themes and have found that there aren’t references to the tabline, although I could be wrong.

**Comment by jinh0** [comment]
What do you mean when you say editing the airline tabline? Do you mean these lines?
```
" tabline colors
let s:tab_fill = [ "#ecf0c1", "#0f111b", 255, 233]
let s:selected_tab = [ "#b3a1e6", "#0f111b", 146, 233 ]
let s:modified_tab = [ "#5ccc96", "#0f111b", 78, 233 ]
let s:background_tab = [ "#686f9a", "#0f111b", 60, 233 ]

" reference airline-tabline-hlgroups for more information
let g:airline#themes#spaceduck#palette.tabline = {
  \ "airline_tab": s:background_tab,
  \ "airline_tab_right": s:background_tab,
  \ "airline_tabsel": s:selected_tab,
  \ "airline_tabsel_right": s:selected_tab,
  \ "airline_tabmod": s:modified_tab,
  \ "airline_tabmod_right": s:modified_tab,
  \ "airline_tabmod_unsel": s:modified_tab,
  \ "airline_tabmod_unsel_right": s:modified_tab,
  \ "airline_tabfill": s:tab_fill,
  \ "airline_tablabel": s:background_tab,
  \ "airline_tablabel_right": s:background_tab,
  \ "airline_tabhid": s:background_tab,
  \ "airline_tabhid_right": s:background_tab
  \ }
```

I made these changes in the `autoload/airline/themes/spaceduck.vim`. I implemented it seeing how [gruvbox did it](https://github.com/morhetz/gruvbox/blob/master/autoload/airline/themes/gruvbox.vim). I'm sorry I'm a little confused at what you mean (I'm new to all this colorscheme stuff).

**Comment by pineapplegiant** [comment]
Gotchaa.
No I am too don’t worry, I’m not sure what I’m asking either. Let me play with this tonight and then see if I can get it merged either tonight or tomorrow.

**Comment by pineapplegiant** [comment]
Switched this to dev homie. Super humble to have this be your first contribution on Github 🚀

**Review by pineapplegiant — APPROVED** [review]
I had a commit leftover for when I was changing the readme, but it should be okay.

**Pull request merged**